### DISCUSSION TITLE
Is the name of the project related to the Tolkien's Gollum?

### DISCUSSION THREAD

**Discussion by Rashair** [gollum/gollum#1757] [General]
If so, how? I couldn't find any information on that :/

**Comment by dometto** [comment]
I think the answer is: almost certainly! Look at the [org's icon](https://github.com/gollum) and the [test wiki](https://github.com/gollum/gollum/tree/master/test/examples/lotr.git). :) But I have no idea why the project was named the way it is. I think it has always been called `gollum`, back from when it was first developed at GitHub. We have only inherited it later.
